Client/Frontend/Widgets/SnackBar.swift

```diff
@@ -177,4 +171,14 @@ class SnackBar: UIView {
             snackButton.drawSeparator()
         }
     }
+
+    // MARK: - Theme Applicable
+    func applyTheme(theme: Theme) {
+        let colors = theme.colors
+        buttonsView.distribution = .fillEqually
```

Comment:
Stackview distribution shouldn't be part of the `applyTheme`. 

It can be done by using the `.build like so:
```
    private lazy var buttonsView: UIStackView = .build { stackView in
        stackView.distribution = .fillEqually
    }
```